packages/jest-core/src/watch.ts

```diff
@@ -228,6 +232,36 @@ export default async function watch(
   let shouldDisplayWatchUsage = true;
   let isWatchUsageDisplayed = false;
 
+  // Initialize Vite dev server manager for watch mode
+  let viteDevServerManager: ViteDevServerManager | undefined;
+  if (contexts.length > 0 && isViteEnabled(contexts[0].config)) {
+    const viteConfig = resolveViteConfig(contexts[0].config);
+    viteDevServerManager = new ViteDevServerManager(
```

Comment:
In runJest.ts, we also initialize an instance of dev server and here we also initialize another one. This is not the right way. We should only use single global instance of Vite